GVFS/GVFS.Installer.Mac/scripts/Distribution_GVFS.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename file `Distribution_GVFS.xml` -> `Distribution_VFSForGit.xml`

Reply:
Consolidated the two separate `xml`s into one. The new one has placeholders for `VFSForGit` and `Git` package info. The placeholders are substituted with actual values (like `VFSForGit` version, package name etc) during build time.